File: abc/abc193/a.cpp
// https://atcoder.jp/contests/abc193/submissions/25679191

#include <bits/stdc++.h>
using namespace std;

int main() {
  int A, B;
  cin >> A >> B;
  double ans = (double)B / (double)A * 100;
  cout << 100 - ans;
}

File: abc/abc193/a.cs
// https://atcoder.jp/contests/abc193/submissions/46508353

using System;

public class A
{
  static void Main()
  {
    string[] t = Console.ReadLine().Split(" ");
    double a = double.Parse(t[0]);
    double b = double.Parse(t[1]);

    Console.Write(100 - b / a * 100);
  }
}

